### PR TITLE
translated warnings/invalid-aria-prop

### PR DESCRIPTION
**Pull request by mohshbool**
done translating warnings/invalid-aria-prop.*

### CODE REVIEW

File: content/warnings/invalid-aria-prop.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ layout: single
 permalink: warnings/invalid-aria-prop.html
 ---
 
-The invalid-aria-prop warning will fire if you attempt to render a DOM element with an aria-* prop that does not exist in the Web Accessibility Initiative (WAI) Accessible Rich Internet Application (ARIA) [specification](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties).
+تحذير 'invalid-aria-prop' سَيُطلَق إذا قُمت بمُحاولة تصيير عنصر DOM مع خاصية '*-aria' غير موجودة في [مواصفات](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties) مبادرة إتاحة الويب "Web Accessibility Initiative WAI" و تطبيق انترنت متاح غني "Accessible Rich Internet Application ARIA".
```
Comment (mohshbool) [inline content/warnings/invalid-aria-prop.md]:
```suggestion
تحذير 'invalid-aria-prop' سَيُطلَق إذا قُمت بمُحاولة تصيير عنصر DOM مع خاصية '*-aria' غير موجودة في [مواصفات](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties) "Web Accessibility Initiative WAI" و "Accessible Rich Internet Application ARIA".
```

Comment (mohshbool) [reply content/warnings/invalid-aria-prop.md]: @Fcmam5 could you take a look at this? Thanks :)

Comment (Fcmam5) [reply content/warnings/invalid-aria-prop.md]: That's better as you translated it, like we translate to words/names if it's possible and we add the original name between parentheses. Good job!

File: content/warnings/invalid-aria-prop.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ layout: single
...
 
-1. If you feel that you are using a valid prop, check the spelling carefully. `aria-labelledby` and `aria-activedescendant` are often misspelled.
+1. تأكَد من الإملاء بحذر ان كُنت تَظُنّ انك تستخدم خاصية صالحة. خاصيتا `aria-labelledby` و `aria-activedescendant` عادةً ما يُخطَأُ بِتَهجِئَتِهما.
 
-2. React does not yet recognize the attribute you specified. This will likely be fixed in a future version of React. However, React currently strips all unknown attributes, so specifying them in your React app will not cause them to be rendered
\ No newline at end of file
+2. لا تَتعرَّف React على الخاصية التي حَدَدتَها. سيتم إصلاح ذلك في إصدار مستقبلي من React على الأرجح. تُزيل React حاليُا جميع الخواص غير المُعَرَّفة ، لذلك تحديدهم في تطبيق React خاصَتِك لن يؤدي إلى تَصييرِهم.
```
Comment (Fcmam5) [inline content/warnings/invalid-aria-prop.md]: Fix teshkeel :), we always have that problem between ُ ً َ they are too close to each other in our keyboards :smile:, I had to zoom my editor & browser to verify such tricky things.

```suggestion
2. لا تَتعرَّف React على الخاصية التي حَدَدتَها. سيتم إصلاح ذلك في إصدار مستقبلي من React على الأرجح. تُزيل React حاليًا جميع الخواص غير المُعَرَّفة ، لذلك تحديدهم في تطبيق React خاصَتِك لن يؤدي إلى تَصييرِهم.
```

Comment (mohshbool) [reply content/warnings/invalid-aria-prop.md]: Welp, this is not outdated but I think this is what you were going for, right?

```suggestion
2. لا تَتعرَّف React على الخاصية التي حَدَدتَها. سيتم إصلاح ذلك في إصدار مستقبلي من React على الأرجح. تُزيل React حاليًا جميع الخواص غير المُعَرَّفة ، لذلك تحديدهم في تطبيق React خاصَتِك لن يؤدي إلى تَصييرِهم.
```

**Comment by iRayan7** [comment]
Hi @Fcmam5, please review this PR and merge it after approving it.
make sure you checked out this note from #1
> Warnings
> These are the pages that you get when you click the links in the console (e.g. https://reactjs.org/warnings/dont-call-proptypes.html). People tend to search these, so please don't translate the error message itself.

Thanks.

File: content/warnings/invalid-aria-prop.md (hunk 2 above)
Comment (Fcmam5) [reply content/warnings/invalid-aria-prop.md]: Sure, you are the boss, sir! :smiley:

File: content/warnings/invalid-aria-prop.md [hunk 3]
```diff
@@ -4,8 +4,8 @@ layout: single
...
+تحذير 'invalid-aria-prop' سَيُطلَق إذا قُمت بمُحاولة تصيير عنصر DOM مع خاصية '*-aria' غير موجودة في [مواصفات](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties) مبادرة إتاحة الويب "Web Accessibility Initiative WAI" وتطبيق إنترنت متاح غني "Accessible Rich Internet Application ARIA".
 
-1. If you feel that you are using a valid prop, check the spelling carefully. `aria-labelledby` and `aria-activedescendant` are often misspelled.
+1. تأكَد من الإملاء بحذر ان كُنت تَظُنّ انك تستخدم خاصية صالحة. خاصيتا `aria-labelledby` و `aria-activedescendant` عادةً ما يُخطَأُ بِتَهجِئَتِهما.
 
-2. React does not yet recognize the attribute you specified. This will likely be fixed in a future version of React. However, React currently strips all unknown attributes, so specifying them in your React app will not cause them to be rendered
\ No newline at end of file
+2. لا تَتعرَّف React على الخاصية التي حَدَدتَها. سيتم إصلاح ذلك في إصدار مستقبلي من React على الأرجح. تُزيل React حاليُا جميع الخواص غير المُعَرَّفة ، لذلك تحديدهم في تطبيق React خاصَتِك لن يؤدي إلى تَصييرِهم.
```
Comment (mohshbool) [inline content/warnings/invalid-aria-prop.md]:
```suggestion
2. لا تَتعرَّف React على الخاصية التي حَدَدتَها. سيتم إصلاح ذلك في إصدار مستقبلي من React على الأرجح. تُزيل React حاليًا جميع الخواص غير المُعَرَّفة ، لذلك تحديدهم في تطبيق React خاصَتِك لن يؤدي إلى تَصييرِهم.
```

**Comment by mohshbool** [comment]
@Fcmam5 I think it's done now 😄

**Pull request merged**